Sources/__SwiftOnServer_org/Documentation.docc/2024/03/26-getting-started-with-structured-concurrency-in-swift/26-getting-started-with-structured-concurrency-in-swift.md

```diff
@@ -547,10 +530,10 @@ taskGroup.addTask(executorPreference: executor) {
 }
 ```
 
-As you may notice, the `EventLoopExecutor` type is _manually_ retained and released. This is becasue the `addTask` method does not retain the executor. If the EventLoopExecutor type is not retained elsewhere, it will be deallocated before the task is done running, causing a crash.
+As you may notice, the `EventLoopExecutor` type is _manually_ retained and released. This is becasue the ``TaskGroup.addTask`` method does not retain the executor. If the EventLoopExecutor type is not retained elsewhere, it will be deallocated before the task is done running, causing a crash.
 
 #### Running Heavy Workloads
 
-Previously, we wrote that large workloads should be run outside of structured concurrency. This is necessary, since the _standard_ executor in Swift is designed to run tasks concurrently. In Swift 6, this is executor is the `globalConcurrentExecutor`, which is hidden in previous versions of Swift.
+Previously, we wrote that large workloads should be run outside of structured concurrency. This is necessary, since the _standard_ executor in Swift is designed to run tasks concurrently. In Swift 6, this is executor is the ``globalConcurrentExecutor``, which is hidden in previous versions of Swift.
```

Comment:
this link will not work until Swift 6 is released, as [SE-0417](https://github.com/apple/swift-evolution/blob/main/proposals/0417-task-executor-preference.md) did not make it into 5.10